### PR TITLE
DOC-539: Named Aggregations

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR reworks the Named Aggregations topic.

- Moves them out of queries and into a new section. @StefanMacrometa - Should these actually be within queries? 
- Breaks up the topic.
- Minor format and grammar edits.

### CODE REVIEW

File: docs/cep/aggregations/distributed-aggregations.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+|Item | Description|
+|------|------|
+@partitionById | If the property is given, then the distributed aggregation is enabled. This can be disabled by using `enable` element, `PartitionById.enable='false'`.|
+
+## System Properties
+
+System Property| Description| Possible Values | Optional | Default Value
+---------|---------|---------|---------|------
+shardId| The ID of the shard one of the distributed aggregation is running in. This should be unique to a single shard. | Any string | No | <Empty_String>
+partitionById| This allows user to enable/disable distributed aggregation for all aggregations running in one stream processing manager. | true/false | Yes | false
```
Comment (diana-macrometa) [inline docs/cep/aggregations/distributed-aggregations.md:29]: @StoyanTc, by "stream processing manager", do they mean stream worker here?

File: docs/cep/aggregations/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+from <input stream>
+    group by <attribute name>
+    aggregate by <timestamp attribute> every <time periods> ;
+```
+
+## Parameters
+
+| Item    | Description |
+|---------------|---------|
+| store                      | Used to refer to the data store where the calculated aggregate results are stored. This annotation is optional. When no annotation is provided, the data is stored in the `in-memory` store. |
```
Comment (diana-macrometa) [inline docs/cep/aggregations/index.md:35]: @StoyanTc, is this bit about in-memory store correct?

File: docs/cep/aggregations/distributed-aggregations.md (hunk 1 above)
Comment (StoyanTc) [reply docs/cep/aggregations/distributed-aggregations.md:29]: @diana-macrometa - No, this is the manager of all stream workers.

**Review by StoyanTc — APPROVED** [review]

File: docs/cep/aggregations/index.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/cep/aggregations/index.md:35]: @StoyanTc, please answer this before I merge.

Comment (StoyanTc) [reply docs/cep/aggregations/index.md:35]: @diana-macrometa - Yes, that's correct. Every aggregation keeps the actual data in memory.

**Pull request merged**